components/atom/tag/src/index.scss

```diff
@@ -0,0 +1,73 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$bgc-AtomTag: color-variation($c-gray, 3);
+$c-AtomTagActionable: $c-white;
+$h-AtomTag: $p-xxl;
+$mw-label: 240px;
+
+.sui-AtomTag {
+  background-color: $bgc-AtomTag;
+  border-radius: ceil($h-AtomTag / 2);
+  box-sizing: border-box;
+  display: inline-block;
+  height: $h-AtomTag;
+  line-height: ceil($h-AtomTag / 2);
+  padding: $p-m $p-l;
+  white-space: nowrap;
+
+  & + & {
+    margin-left: $m-m;
+  }
+
+  &-label {
+    // max width - 2 icon size
+    $w-label-base: $mw-label - ($p-l * 2);
+    display: inline-block;
+    max-width: $w-label-base;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+
+    // if only one icon is set
+    .sui-AtomTag-hasIcon &, .sui-AtomTag-hasClose & {
+      max-width: $w-label-base - ($p-l + $m-s);
+    }
+
+    // if both icons are set
+    .sui-AtomTag-hasIcon.sui-AtomTag-hasClose & {
+      max-width: $w-label-base - ($p-l + $m-s) * 2;
+    }
+  }
+
+  &-icon {
+    margin-right: $p-s;
+  }
+
+  &-secondary-icon {
+    cursor: pointer;
+    margin: 0 (-$m-s) 0 $m-m;
+  }
```

Comment:
a negative margin is necessary?  (-$m-s)

Reply:
After talking about it "in person" with @beatrizip,  it's ok!